include/rangers-api/Application/Resource/AppResourceManager.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+namespace app::fnd {
+    class AppResourceManagerListener {
```

Comment:
Starting braces should be started on a line to keep consistent with the original repository.